setup.sh

```diff
@@ -126,14 +126,6 @@ if ! hash ansible-playbook &> /dev/null; then
     pip install --upgrade --editable git+https://github.com/ansible/ansible.git@devel#egg=ansible
 fi
 
-# User doesn't have a default virtualenv, let's configure one
-if [ ! -f ~/.bashrc ] || ! grep 'source.*activate' ~/.bashrc; then
-    echo '# Activate ansible virtualenv' >> ~/.bashrc
-    echo 'source ~/.ansible-env/bin/activate' >> ~/.bashrc
-    echo 'export ANSIBLE_STDOUT_CALLBACK=debug' >> ~/.bashrc
-    echo '!! You need to login again for changes to take effect !!'
-fi
```

Comment:
I don't think that's enough, forcing non-python hackers to take an additionnal action that can easily be automated is error prone and definitely is not the solution here.

I've already seen things like, so it's not started here, but then again, I simply fail to see how that argument can be constructive, but I trust you it will, so I'm answering :)